package.json

```diff
@@ -110,7 +109,7 @@
       "<rootDir>/src/**/*.{spec,test}.{js,jsx,ts,tsx}"
     ],
     "testEnvironment": "jsdom",
-    "testRunner": "/Users/mamartynov/Projects/jest-resolve-bug/node_modules/jest-circus/runner.js",
+    "testRunner": "jest-circus/runner.js",
```

Comment:
bug in CRA, I reported it here: https://github.com/facebook/create-react-app/issues/9993